02-express-tutorial/final/07-params-query.js

```diff
@@ -1,58 +1,61 @@
-const express = require('express')
-const app = express()
-const { products } = require('./data')
+const express = require('express');
```

Comment:
I think the changes in this file should have been made in the `answers` directory instead